rfcs/text/0000-roadmaps+planning.md

```diff
@@ -0,0 +1,421 @@
+
+**Note**: *Input from the Technical Architecture team is integrated on a quarterly basis during Release planning.*
+
+Day 1 in the Roadmap timeline begins in the third quarter.
+
+* **Day 1 - Day 25: [Gathering and reviewing input](#gathering-and-reviewing-input)**
+  * Prerequisite: Science Roadmap
+  * Day 1-25: DCP User Survey 
+  * Day 1-10: Roadmap Retrospective
+* **Day 1 - Day 40: [Drafting the roadmap](#drafting-the-roadmap)**
+  * Day 1-10: Identify a Roadmap Shepherd
+  * Day 25-30: Create themes
+  * Day 31: Collate themes
+  * Day 32-40: Refine themes
+* **Day 40 - Day 55: Receiving feedback**
+  * Day 40: Publish draft Roadmap
+  * Day 41-50: Community Review period 
+  * Day 51-55: Complete final edits
+* **Day 55 - Day 65 : [Iterating with Oversight](#iterating-with-oversight)**
+  * Day 55: Send Roadmap to Oversight
+  * Day 56-65: Review and edit roadmap with Oversight
+  * Day 65: Receive Roadmap approval from Oversight
+  * Day 65: Communicate approved Roadmap 
+
+### Gathering and Reviewing Input
+[Gathering and Reviewing Input]:#gathering-and-reviewing-input
+
+#### Science Roadmap
+
+Oversight MUST provide an annual Science Roadmap to Project Leads to be reviewed at the next Oversight meeting following its receipt.
+
+Project Leads MUST clarify outstanding questions about the Science Roadmap with Oversight within two weeks.
+
+#### Technical Architecture Objectives 
+
+Through a process of their own choosing, the Technical Architecture team MAY document critical architectural issues and technical debt in a Technical Objectives Informational RFC for consideration during each Release. The RFC SHOULD outline technical strategies and estimate cost (such as two engineers at 20%) to realize its objectives. 
+
+The RFC MUST be approved prior to [Clarifying the Next Release](#clarifying-the-next-release) for consideration.
+
+#### DCP User Survey
+
+User Experience (UX) MUST create a DCP user survey on an annual basis to collect input for the Product Roadmap. The audience for the survey are the Users of the DCP.
+
+UX MUST review the proposed survey with the DCP community and incorporate suggestions as appropriate.
+
+The survey MUST be announced to the membership of the Human Cell Atlas (through its the Registration mailing list) with a minimum of a two week last call for responses.
+
+At the conclusion, UX MUST summarize insights from the survey and present at the next DCP PM meeting. Following a one week review to incorporate feedback, the survey MUST be shared by UX with the membership of the Human Cell Atlas and announced on the *HumanCellAtlas #dcp* slack channel.
+
+#### Roadmap Retrospective
+
+Project Leads MUST independently review the prior year’s Product Roadmap, Release Retrospective summaries, and appropriate supporting documentation.
+
+Project Leads MUST share their significant findings and learnings during the Retrospective. 
+
```

Comment:
It's not clear to me who the PLs are sharing their findings/ learnings with?  With each other?  With the rest of the DCP team?  With the wider HCA community?

Reply:
@ambrosejcarr My plan was for PLs to share with one another. If it makes sense to you and @adriennes , we can have it be open to the community. I don't have a preference either way, and my main focus was on ensuring that we took a minute and looked back at what needs to change moving forward.

Reply:
I'd vote for open to community, but open to push-back from others here if they can think of ways that this would diminish the value of the session.

Reply:
I'd recommend a small Project Lead-only retrospective and the conclusions shared with the community. And if it's open, the community is invited as optional to make scheduling easier.

Reply:
Clarified in 602124f5ab4a0fa2846c6d04409702c8e20c2df7 - @CaraLMason and I discussed and agreed that this should be a closed session to allow the Project Leads to have an open and honest conversation in private.

Reply:
Supportive of the closed session -- but I think that the PLs should generate some type of read-out/artifact to share with the DCP Community/Oversight. A set of "lessons learned" seems to be helpful only if it's summarized and shared.

Reply:
Agreed, focused session with transparency. There are many spaces for feedback to be given from the team, this being just PLs does not block feedback or ideas being pulled into the roadmap.

Reply:
As part of the rewrite, there will be a common Retrospective led by the Project Leads for both roadmap and planning+execution - https://github.com/HumanCellAtlas/dcp-community/pull/92/commits/8c3d1426aa4eb608aad529590428f1ae1f2f88a8. I feel that it's important to have broader awareness and appreciation of the challenges on both ends of the process. With that in mind, I'm going to resolve this conversation.